model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,47 @@
+groups:
+  - id: mobile-crash-event
+    prefix: hello
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash'.
+    attributes:
+      - ref: ios.state
+        requirement_level:
+          recommended: If the crash occurred on an iOS device.
```

Comment:
`ios` is specifically for iPhone, `macOS` is for MacBook and friends.
Maybe we call it `apple` instead? since the crash reports share the same format on all Apple platforms.